src/components/molecules/AdminInput/AdminInput.tsx

```diff
@@ -5,7 +5,7 @@ import { get } from "lodash";
 
 import { generateId } from "utils/string";
 
-import "./AdminInput.scss";
+import CN from "./AdminInput.module.scss";
```

Comment:
Probably should remove the SCSS module and create `AdminInput.tailwind.ts` file, possibly as a new component in a new place.